cdap-proto/src/main/java/co/cask/cdap/proto/BatchProgramCount.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.proto;
+
+import java.util.Objects;
+import javax.annotation.Nullable;
+
+/**
+ * Result for the program count
+ */
+public class BatchProgramCount extends BatchProgramResult {
+  private final int count;
+
+  public BatchProgramCount(BatchProgram program, int statusCode, @Nullable String error, int count) {
+    super(program, statusCode, error);
+    this.count = count;
+  }
+
+  /**
+   * @return the count of the program. -1 if there is an error
```

Comment:
nit: `count of program run`. If this is -1

Reply:
fixed